### ISSUE TITLE
Show the existing authenticated user (OAuth)

### ISSUE THREAD

**Issue by nataliekwong** [airbytehq/airbyte#31941]
Often when I'm trying to debug a connection issue and think it might be a permissions issue, I will try to first check who authenticated the connection originally. It will only show me that the source is authenticated, but it won't show me what the username/authentication is using. 

It would be great to see this so I don't have to guess. The workaround is to re-authenticate again and hope that my new authentication is better than the last one and my sync will succeed on the next run.

![Screenshot 2023-10-27 at 3 06 21 PM](https://github.com/airbytehq/airbyte/assets/38087517/94c205d0-f1f6-464c-b1e2-888ebb0089e0)

**Comment by nataliekwong** [comment]
Example from Segment:
<img width="779" alt="Screenshot 2023-11-01 at 3 47 58 PM" src="https://github.com/airbytehq/airbyte/assets/38087517/67e24d11-1cac-4053-9f92-c75658f2a7aa">

**Comment by katmarkham** [comment]
@lmossman this also seems like a really nice feature.  We could write a support article on it.  It probably comes up more than we expect.  @nataliekwong do we have any way of seeing how many accounts have previously changed credentials in the past?

**Comment by nataliekwong** [comment]
I don't think we can report on that, we don't store the credentials anywhere (AFAIK)
The closest reporting we have is that 5% of users are clicking "Re-authenticate": https://app.fullstory.com/ui/13AXQ4/metrics/details/cHdvBE1E4hCt?completeSessions=false&event=0

It can't differentiate if they are truly re-authenticating with a new user or are just re-testing the existing authentication.
